portal-web/test/functional/com/liferay/portalweb/paths/portlet/claysample/ClaySamplePortlet.path

```diff
@@ -106,6 +106,17 @@
 	<td>//section[contains(@id,'portlet_ClaySamplePortlet')]</td>
 	<td></td>
 </tr>
+<!--MANAGEMENT_TOOLBARS-->
+<tr>
+	<td>DISPLAY_CONTEXT_CHECKBOX</td>
+	<td>//div[not(contains(@class,'hide')) and contains(@id, 'tabs')]//h3[text()='USING DISPLAY CONTEXT']//following::nav[contains(@class,'management-bar')]//input[@type='checkbox']</td>
```

Comment:
From the way I saw Clay Sample portlet's html structure, I have a feeling this could target other html elements. `following::nav` will target all other siblings and children not just the immediate sibling. This would probably work for now but has potential to refine with something like `following::nav[ ... ][1]` to target the immediate next one.